combat-trainer.lic

```diff
@@ -1171,6 +1181,15 @@ class AbilityProcess
         next unless !timer || (Time.now - timer).to_i > 30
         game_state.cooldown_timers[name] = Time.now if activate_khri?(@kneel_khri, name)
       end
+    @barb_buffs.each do |name|
+      if !(Flags["ap-#{name}-active"] == true)
```

Comment:
So, no. I had it !Flags before, but when I lose a form, if you print out the Flag it isn't a boolean, it becomes the flag's reset data. 
```
[barb-buff-ct: INSIDE BUFF LOOP: name=Python -- FLAG: ["finish practicing the Form of the Python"]]
[barb-buff-ct: INSIDE BUFF LOOP: name=Piranha -- FLAG: true]
[barb-buff-ct: INSIDE BUFF LOOP: name=Eagle -- FLAG: true]
[barb-buff-ct: INSIDE BUFF LOOP: name=Dragon -- FLAG: ["finish practicing the Form of the Dragon"]]
[barb-buff-ct: INSIDE BUFF LOOP: name=Monkey -- FLAG: true]
[barb-buff-ct: INSIDE BUFF LOOP: name=Tornado -- FLAG: true]
[barb-buff-ct: INSIDE BUFF LOOP: name=Earthquake -- FLAG: true]
```

The way to go around this is to do what I did. Can't trigger off False.